kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/adapters/KmVideoRMAdapter.java

```diff
@@ -72,7 +73,7 @@ void bindItems(RecyclerView.ViewHolder viewHolder, int position) {
         if (payloadList != null) {
             final KmRichMessageModel.KmPayloadModel payloadModel = payloadList.get(position);
             WebSettings webSettings = holder.webview.getSettings();
-            webSettings.setJavaScriptEnabled(true);
+            webSettings.setJavaScriptEnabled(new AlCustomizationSettings().isJavaScriptEnabled());
```

Comment:
AlCustomizationSettings cannot be initialized like this, see other initialization of AlCustomizationSettings.
You can pass th ealCustomzationSettings wherever it's initialized to this Adapter class.

Reply:
Changed as suggested